generate.xml

```diff
@@ -80,6 +80,8 @@
       </product>
       <product prefix="lib" path="src" name="bitcoin" uuid="39f60708-ff48-4c22-952d-43470866f684" >
         <flag substitute="icu" />
+        <flag substitute="png" />
+        <flag substitute="qrencode" />
```

Comment:
I'm pretty sure that these explicit flag declarations are only required for libbitcoin `<product>` elements. This is because a build doesn't use its own package config (e.g. for test/console/example). The other products rely on the flag as published by the libbitcoin package config. Basically the dependent packages are guaranteed through package config to get the flags that libbitcoin was built with. So you should remove the <flag> elements in the non-libbitcoin repositories.